modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/create_devices.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module DhpConnectedDevices
+  module DataMigrations
+    module CreateDevices
+      module_function
+
+      def run
+        Device.create(key: 'fitbit', name: 'Fitbit')
```

Comment:
Does this only run once? / what happens after the first time it's created?

Reply:
The rake task can be run more than once (it will not create a new record if a device with the give key already exists)

Reply:
Will it throw an error / does it matter if it throws an error?

Reply:
good question. I don't think so, but can update it to be an upsert

Reply:
Updated to get (if exists) or create